pkg/adminconfig/evaluator_interfaces.md

```diff
@@ -0,0 +1,135 @@
+Component for Config Policy Decisions
+==========================
+
+## Summary
+
+Design a component that will evaluate config policy decisions based on the fybrikapplication spec and infrastructure (e.g. deployed clusters).
```

Comment:
can you better describe the input and output. e.g., received the workload information in fybrikapplication spec and ... a d return the decisions? tha manager needs to implement.
Or something along these lines? should be more readable and easier to understand.